packages/server/src/controllers/accounts.ts

```diff
@@ -5,23 +5,18 @@ import { extractSessionId } from "../middlewares/authenticate";
 import { getAccountBySessionId } from "../db/sessions";
 
 export const createAccount = async (ctx: Context) => {
-  try {
-    const { firstName, lastName, email, password } = ctx.request.body;
-    const hashedPassword = await hashPassword(password);
+  const { firstName, lastName, email, password } = ctx.request.body;
+  const hashedPassword = await hashPassword(password);
 
-    await createOneAccount(firstName, lastName, email, hashedPassword);
-    ctx.body = {
-      status: "ok",
-      account: {
-        firstName,
-        lastName,
-      },
-    };
-    ctx.status = 201;
-  } catch (error) {
-    ctx.status = 500;
-    ctx.message = "Error when creating account";
-  }
```

Comment:
Why removing the error handling ? :))

Reply:
It was simpler for debugging. I suggest to never catch errors and only handle them at the end. It is done in index.ts: 
app.on("error", (err) => {
  console.log(err);

Reply:
Mmmh not sure it's that great for logging/monitoring purposes + security to just log the error in the index level.
What I can do is leave all the error catching in the handler level but add a log of the error object with serialized stack trace for debugging purposes :))

Reply:
Indeed, we need a proper way of handling errors. 
I have put back the try-catch.